nats/server_report_command.go

```diff
@@ -339,18 +341,23 @@ func (c *SrvReportCmd) reportAccount(_ *kingpin.ParseContext) error {
 	}
 
 	table := newTableWriter(fmt.Sprintf("%d Accounts Overview", len(accounts)))
-	table.AddHeaders("Account", "Connections", "In Msgs", "Out Msgs", "In Bytes", "Out Bytes", "Subs")
+	table.AddHeaders("Account", "Server/Cluster", "Connections", "In Msgs", "Out Msgs", "In Bytes", "Out Bytes", "Subs")
 
 	for _, acct := range accounts {
-		table.AddRow(acct.Account, humanize.Comma(int64(acct.Connections)), humanize.Comma(acct.InMsgs), humanize.Comma(acct.OutMsgs), humanize.IBytes(uint64(acct.InBytes)), humanize.IBytes(uint64(acct.OutBytes)), humanize.Comma(int64(acct.Subs)))
+		server := []string{}
+		for _, s := range acct.Server {
+			server = append(server, fmt.Sprintf("%s/%s", s.Name, s.Cluster))
```

Comment:
Is cluster always set? I think we generate it right so probably is

Reply:
fixed